tests/bearlib/abstractions/LinterTest.py

```diff
@@ -180,7 +211,10 @@ def test_decorator_invalid_parameter_types(self):
                          "The value 'invalid' for key 'critical' inside given "
                          'severity-map is no valid severity value.')
 
-        with self.assertRaises(TypeError) as cm:
+        with self.assertRaisesRegex(TypeError,
+                                    'Invalid severity value 389274234 for key '
+                                    "'critical-error' inside given severity-"
```

Comment:
Just for consistency, you can use single quote and escape with "\\" wherever required :)